### PR TITLE
Release notes draft for 6.8.9

### PR DESCRIPTION
**Pull request by logstashmachine**
**PREVIEW:**  http://logstash_11851.docs-preview.app.elstc.co/guide/en/logstash/6.8/releasenotes.html

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -40,6 +41,30 @@ This section summarizes the changes in the following releases:
...
+bf3282cfa - performance: share a single secret store (3 weeks ago) <Ry Biesemeyer>
+00f6f3b34 - Version bump to 6.8.9 (#11734) (4 weeks ago) <Rob Bavey>
+30031c8c5 - Release Notes for 6.8.8 (#11704) (5 weeks ago) <Rob Bavey>
+
+=== Logstash Plugin Release Changelogs ===
+Computed from "git diff v6.8.8..6.8 *.release"
+Changed plugin versions:
+---------- DELETE UP TO HERE ------------
+
+==== Plugins
```
Comment (colinsurprenant) [inline docs/static/releasenotes.asciidoc:66]:
```suggestion
* Performance: share a single secret store https://github.com/elastic/logstash/pull/10794[#10794]

```

**Comment by colinsurprenant** [comment]
There isn't much other changes. https://github.com/elastic/logstash/compare/v6.8.8...6.8

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -40,6 +41,13 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-8-9]]
+=== Logstash 6.8.9 Release Notes
+
+* Performance: Share a single secret store to reduce overhead for pipelines that use these variables. 
+We no longer incur the cost of repeatedly building the keystore. https://github.com/elastic/logstash/pull/10794[#10794]
```
Comment (yaauie) [inline docs/static/releasenotes.asciidoc:48]:
```suggestion
* * Performance: improved startup- and reload-time for pipelines with many variable subsitutions when a secret store is present.
We no longer incur the cost of rebuilding the keystore for each substitution.
https://github.com/elastic/logstash/pull/10794[#10794]
```

**Comment by yaauie** [comment]
LGTM 👍

**Pull request merged**